### PR TITLE
DOCS 1484: Update DB Extractor schema JSON

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: schema/cognite_config.schema.json [hunk 1]
```diff
@@ -130,7 +130,7 @@
                 "disable-gzip": {
                     "type": "boolean",
                     "default": false,
-                    "description": "Whether or not to disable gzipping of json bodies."
+                    "description": "Set to true to disable gzipping of json bodies."
```
Comment (mathialo) [inline schema/cognite_config.schema.json:133]:
``` suggestion
                    "description": "Set to `true` to disable gzipping of json bodies."
```

File: schema/cognite_config.schema.json [hunk 2]
```diff
@@ -139,34 +139,34 @@
...
                     "type": "integer",
                     "default": 50,
-                    "description": "The maximum number of connections which will be kept in the SDKs connection pool."
+                    "description": "The maximum number of connections in the SDKs connection pool."
                 },
                 "disable-ssl": {
                     "type": "boolean",
                     "default": false,
-                    "description": "Whether or not to disable SSL verification."
+                    "description": "Set to true to disable SSL verification."
```
Comment (mathialo) [inline schema/cognite_config.schema.json:162]:
``` suggestion
                    "description": "Set to `true` to disable SSL verification."
```

File: schema/metrics_config.schema.json [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
     "type": "object",
     "properties": {
         "push-gateways": {
             "type": "array",
-            "description": "List of prometheus pushgateway configurations",
+            "description": "List of Prometheus pushgateway configurations",
             "items": {
                 "type": "object",
-                "description": "The push-gateways sections contain a list of metric destinations.",
+                "description": "The pushgateways sections contain a list of metric destinations.",
```
Comment (mathialo) [inline schema/metrics_config.schema.json:12]: This should be hyphenated, it's the name of a config parameter, and the extractors will not recognize it as one word
``` suggestion
                "description": "The `push-gateways` sections contain a list of metric destinations.",
```

File: schema/metrics_config.schema.json [hunk 4]
```diff
@@ -57,36 +57,36 @@
                 },
                 "asset-name": {
                     "type": "string",
-                    "description": "Enter the name for a CDF asset that will have all the metrics time series attached to it."
+                    "description": "Enter the name for a CDF asset with all the metrics time series attached to it."
```
Comment (mathialo) [inline schema/metrics_config.schema.json:60]: The asset might not yet exist, we will set all the time series on it, which why it was written in the tense it was

File: schema/state_store_config.schema.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
     "$id": "state_store_config.schema.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "description": "Include the state store section to save extraction states between runs. Use this if data is loaded incrementally. We support multiple state stores, but you can only configure one at a time.",
+    "description": "Include the state store section to save extraction states between runs. Use state store if data is loaded incrementally. We support multiple state stores, but you can only configure one at a time.",
```
Comment (mathialo) [inline schema/state_store_config.schema.json:4]:
``` suggestion
    "description": "Include the state store section to save extraction states between runs. Use a state store if data is loaded incrementally. We support multiple state stores, but you can only configure one at a time.",
```

**Review by mathialo — COMMENTED** [review]
Thanks!

Some comments, mostly nitpicks

File: schema/metrics_config.schema.json (hunk 3 above)
Comment (sowmyaraj92) [reply schema/metrics_config.schema.json:12]: @mathialo My bad, I read it as pushgateways.

**Review by mathialo — APPROVED** [review]

**Pull request merged**